src/components/Grid/Column.js

```diff
@@ -17,13 +17,14 @@ const breakPointProp = breakpoint => `${breakpoint}-sizeOf`;
 const sizeRegex = /^([1-9]|1[0-2])-([1-9]|1[0-2])$/;
 
 export const Column = (props, { cssPrefix }) => {
-  const { align, children, className, ...rest } = props;
+  const { align, children, className, omitSldsCol, ...rest } = props;
+
   const prefix = (classes, passThrough) => prefixClasses(cssPrefix, classes, passThrough);
 
-  const sldsClasses = [
-    'col',
-    { [`align-${align}`]: !!align },
-  ];
+  const sldsClasses = [];
```

Comment:
``` js
const sldsClasses = [
  { col: !omitCol },
  { [`align-${align}`]: !!align }
];
```
We are using the `classNames` library here, in order to not having to manipulate the class array with array methods. See above code for an example of usage.